oidc/identity.go

```diff
@@ -42,3 +43,15 @@ func IdentityFromClaims(claims jose.Claims) (*Identity, error) {
 
 	return &ident, nil
 }
+
+// CopyAdditonalClaims copies all additional claims to claims
+func (ident *Identity) CopyAdditonalClaims(claims jose.Claims) {
```

Comment:
1. So additional sanity checks can be added, such as preventing Standard Claims from being copied unintentionally.  This assumes issue #14 is partially addressed so the full list of Standard Claims is known.
1. To prevent downstream systems, such as dex, from each having to implement something similar.

My goal and the origin of this functionality is to be able to copy Additional Claims in a dex connector in 'server/server.go' right before the `user.AddToClaims(claims)` calls.